dataflow/src/org/checkerframework/dataflow/analysis/Store.java

```diff
@@ -54,6 +54,25 @@
     S leastUpperBound(S other);
 
     /**
+     * Compute an upper bound of two stores that is wider than the least upper bound of the two
+     * stores.
```

Comment:
But there must be some advice on when the method is used. Usually it is to jump to a higher abstraction to allow faster termination of a fixed point computation.